internal/app/osd/internal/reg/reg.go

```diff
@@ -160,27 +161,59 @@ func (r *Registrator) Restart(ctx context.Context, in *osapi.RestartRequest) (*o
 	return &osapi.RestartResponse{}, nil
 }
 
-// Dmesg implements the osapi.OSDServer interface. The klogctl syscall is used
-// to read from the ring buffer at /proc/kmsg by taking the
-// SYSLOG_ACTION_READ_ALL action. This action reads all messages remaining in
-// the ring buffer non-destructively.
+// Dmesg implements the osapi.OSDServer interface.
+//
+//nolint: gocyclo
```

Comment:
actually not, comments without a space don't show in godoc.  

So you want all the comments which are not actual documentation to be this style (without a space). I can't find the reference today for it :(

Reply:
you can see `nolint` popping up here in godoc: https://godoc.org/github.com/talos-systems/talos/pkg/chunker/file#File.Read

Reply:
but something is not right, as it pops up here: https://godoc.org/github.com/talos-systems/talos/pkg/constants

Reply:
ok, found the reference: https://github.com/golangci/golangci-lint#nolint

```
Use //nolint instead of // nolint because machine-readable comments should have no space by Go convention.
```